### PR TITLE
a few tweaks, one bugfix ('test' children)

### PR DESCRIPTION
**Pull request by mmcgahan**
a couple small tweaks to clean up the mgiration branch

### CODE REVIEW

File: src/forms/Button.jsx [hunk 1]
```diff
@@ -6,35 +6,30 @@ import { Button as SwarmButton } from '@meetup/swarm-components';
...
-
-		if (component !== 'button') {
-			console.warn(
-				'All Swarm UI v2 Button components are button elements. Future iterations will support links as independent components and the logic will be handled here'
-			);
-		}
-
-		return (
-			<SwarmButton {...props} {...other}>
-				Test
```
Comment (mmcgahan) [inline src/forms/Button.jsx]: this is a bug

**Comment by sadafie** [comment]
Aside from the bug fix, why these changes? I believe in another PR, we agreed to keep the class structure and a few other things to minimize the set of changes in MWC. 

I'm fine with the changes in general, but I want to understand why you felt the need to for these changes so that if it is important we are all agreeing to it and applying the same approaches everywhere, including future components.

File: src/forms/Select.jsx [hunk 2]
```diff
@@ -45,7 +42,7 @@ export const SelectInput = (props: Props) => {
 			name={name}
 			error={error}
 			helperText={helperText}
-			{...requiredProps}
+			requiredText={requiredText}
```
Comment (sergeychernyshev) [inline src/forms/Select.jsx]: Looks like you're removing the pattern I was using (maybe due to my misunderstanding of how props work).

What happens if require is `false`? will `requiredText` be set to false? how will it affect the fact that `requiredText` is only [supposed to be a string](https://github.com/meetup/swarm-ui/blob/master/packages/swarm-components/src/Select.jsx#L33)?

**Comment by sadafie** [comment]
@mmcgahan This PR is outdated - should it be updated or closed?

**Comment by mmcgahan** [comment]
closing

**Pull request closed without merging**